src/librbd/ImageCtx.h

```diff
@@ -154,6 +192,9 @@ namespace librbd {
     xlist<operation::ResizeRequest<ImageCtx>*> resize_reqs;
 
     io::ImageRequestWQ *io_work_queue;
+    ImageQosStatusSingleton* qos_status;
+    SafeTimer *m_timer;
```

Comment:
@dillaman, I don't want to mess our adding logic to the existing safertimer singleton. Initially I used that singleton, then I changed like now, I prefer this way a little bit.